views/publications-details.html

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/two-columns.html" %}
+
+{% block main %}
+<article>
+  <h2>Publication description</h2>
+  <ul class="overview-list">
+    <li>{{ publication.title }}</li>
+  </ul>
+
+  <h2>Source</h2>
+  <a href="{{publication.source_url}}">{{publication.source.name}}</a>
```

Comment:
I don't know. I have almost empty DB and there was clear table, but this table has `source_url` field.
If we don't need to display this info, I can remove that link.